src/browser_action/onEachInteraction.js

```diff
@@ -0,0 +1,42 @@
+export function onEachInteraction(callback) {
+	const valueToRating = score => score <= 200 ? 'good' : score <= 500 ? 'needs-improvement' : 'poor';
```

Comment:
It's possible but I don't think worth jumping through the extra hoops right now. We should maybe consider this mode helpful enough to upstream something like in into web-vitals.js (`onInteraction`?)

Reply:
Let’s wait and see how this lands and how @philipwalton feels. Know he’s been keen on the past on leaving this as a CWV library rather than something more general.

Plus LoAF might change things and adding this to the library would make any change with that a breaking change.

Reply:
Ah, fair, the current logging (especially breakdowns) would have been easier to do elegantly with LoAF.  It may be that that implementation is also a lot less bytes to add to the library.